### PR TITLE
fix: preload libjmkl.so

### LINKED ISSUE #821: Shall we need pre-load MKL?
MKL library will be loaded after one of methods in `MKL.java` is called.
Currently, tensor computing (add, mul) will take some time to pre-load the libjmkl.so at first.

### PR DESCRIPTION
**Pull request by i8run**
## What changes were proposed in this pull request?

Preload libjmkl.so by hand.

## How was this patch tested?

Manual test.

## Related links or issues (optional)
fixed #821

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala [hunk 1]
```diff
@@ -35,6 +35,10 @@ private[tensor] class DenseTensor[@specialized(Float, Double) T: ClassTag](
   var nDimension: Int)(implicit ev: TensorNumeric[T])
   extends Tensor[T] {
 
+  // pre-load MKL library. If we do not do it here,
```
Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala]: why not put it in an object?

Comment (i8run) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala]: If it's in `object DenseTensor`, `new DenseTensor` will not load library automatically.

Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala]: If you put here, each time new DenseTensor, the MKL.isloaded will be run.

If you put in an Tensor object, the MKL.isloaded will be run only when JVM load the class

Comment (jason-dai) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala]: Yes, we should just put it in the initialization code of a Tensor object

**Pull request merged**